flutter/ios/sentry_flutter.podspec

```diff
@@ -15,7 +15,7 @@ Sentry SDK for Flutter with support to native through sentry-cocoa.
   s.dependency 'Sentry/HybridSDK', '8.18.0'
   s.ios.dependency 'Flutter'
   s.osx.dependency 'FlutterMacOS'
-  s.ios.deployment_target = '11.0'
+  s.ios.deployment_target = '12.0'
```

Comment:
This is a breaking change, if intentional, ideally another PR?

Reply:
min version bump to 12 will be part of v8